src/20_Components/amp-install-serviceworker.html

```diff
@@ -24,24 +26,28 @@
 </head>
 <body>
 
-  <!-- #### Basic Usage -->
+  <!-- #### Installing the Service Worker -->
   <!--
-  amp-install-serviceworker should have both properties `src` and `data-iframe-src`; `src` is the location of Service Worker to be installed when the original AMP pages is accessed, `data-iframe-src` is the location of HTML file which calls Service Worker installation process when the cached AMP pages is accessed.
-  The reason why the latter option is required is because browsers can't install Service Worker from other domains for security reasons. For details of the security considerations of Service Worker, [see official specs](https://www.w3.org/TR/service-workers/#security-considerations).
+  `amp-install-serviceworker` should have both properties `src` and `data-iframe-src` set; `src` is the location of service worker to be installed when the original AMP pages is accessed, `data-iframe-src` is the location of HTML file which calls service worker installation process when the cached AMP pages is accessed.
+  The reason why the latter option is required is because browsers can't install service worker from other domains for security reasons. For details of the security considerations of service worker, [see official specs](https://www.w3.org/TR/service-workers/#security-considerations).
```

Comment:
Might make sense to explicitly call out the Google AMP Cache as a cross-origin access point.